app/src/main/java/com/practicum/playlistmaker/player/ui/PlayerActivity.kt

```diff
@@ -99,4 +101,16 @@ class PlayerActivity : AppCompatActivity() {
         }
     }
 
+    private fun clickDebounce(listener: () -> Unit) {
+        if (isClickAllowed) {
+            isClickAllowed = false
+            listener()
+            lifecycleScope.launch { isClickAllowed = true; delay(CLICK_DEBOUNCE_DELAY) }
```

Comment:
```suggestion
            lifecycleScope.launch { 
                delay(CLICK_DEBOUNCE_DELAY) 
                isClickAllowed = true
            }
```
Тут немного некорректно реализован дебаунс, нужно сначала delay вызывать, чтобы отложить присвоение значения `isClickAllowed` на 300мс. Сейчас получается, что как только кликнул - сразу можно ещё раз кликать, без промежутка в 300мс